app/admin/RTMediaFormHandler.php

```diff
@@ -513,6 +571,63 @@ static function general_content() {
 		self::render_tab_content( $render_options, $general_group, 90 );
 	}
 
+	/**
+	 * render export import.
+	 *
+	 * @access static
+	 *
+	 * @return array $render
+	 */
+	static function render_export_import() {
+		$render = array(
+			'rtmedia_export_settings' => array(
+				'title'               => esc_html__( 'Export rtMedia Settings', 'buddypress-media' ),
+				'callback'            => array( 'RTMediaFormHandler', 'button' ),
+				'args'                => array(
+					'id'    => 'rtm-export-button',
+					'key'   => 'rtmedia_export_settings',
+					'value' => esc_html__( 'Export Settings', 'buddypress-media' ),
+					'desc'  => esc_html__( 'This will export rtMedia settings into a JSON file.', 'buddypress-media' ),
+					'class' => array( 'button button-primary button-small' ),
+				),
+				'group'               => 10,
+			),
+			'rtmedia_import_settings' => array(
+				'title'               => esc_html__( 'Import rtMedia Settings', 'buddypress-media' ),
+				'callback'            => array( 'RTMediaFormHandler', 'fileinput' ),
+				'args'                => array(
+					'id'    => 'rtm-import-button',
+					'key'   => 'rtmedia_import_settings',
+					'value' => esc_html__( 'Import Settings', 'buddypress-media' ),
+					'desc'  => esc_html__( 'This will import rtMedia settings. Allowed File Type: json', 'buddypress-media' ),
+				),
+				'group'               => 10,
+				'after_content' => esc_html__('Importing invalid files/settings may break your site. Please import valid file exported from rtMedia only.', 'buddypress-media') ,
+			),
+		);
+
+		return $render;
+	}
+
+
+	/**
+	 * Define rtm_export_import
+	 *
+	 * @access static
+	 *
+	 * @return void
+	 */
+	static function rtm_export_import() {
+		global $rtmedia;
+		$render_options           = self::render_export_import();
+		$render_options           = apply_filters( 'rtmedia_export_import_add_itmes', $render_options );
+		$export_import_group      = array();
+		$export_import_group[10]  = esc_html__( 'Export/Import Settings', 'buddypress-media' );
+		$export_import_group      = apply_filters( 'rtmedia_export_import_groups', $export_import_group );
```

Comment:
Add inline documentation for this filter. Also, add `@since` if this is a new filter.